vendor/bat-native-ledger/src/ledger_impl.cc

```diff
@@ -623,7 +622,9 @@ void LedgerImpl::OnTimer(uint32_t timer_id) {
       std::bind(&LedgerImpl::LoadPublishersListCallback,this,_1,_2,_3));
   } else if (timer_id == last_grant_check_timer_id_) {
     last_grant_check_timer_id_ = 0;
-    FetchGrant(std::string(), std::string());
+    if (!is_android) {
+      FetchGrant(std::string(), std::string(), "");
```

Comment:
why would last_grant_check_timer_id_ ever get set here in the first place? This path doesn't call `RefreshGrant`, correct?

Reply:
if for some reason we do need this, can we change `is_android` to `use_safetynet` or similar?

Reply:
we don't need `is_android` flag at all, i''ll change it

Reply:
Removed all `is_android` flags